api/src/main/java/io/kafbat/ui/config/auth/LdapSecurityConfig.java

```diff
@@ -63,24 +62,39 @@ public ReactiveAuthenticationManager authenticationManager(LdapContextSource lda
       ba.setUserSearch(userSearch);
     }
 
+    AuthenticationManager manager = new ProviderManager(List.of(
```

Comment:
Let's not mix the distinct issues in one PR, please. #54 is to be solved within #717 (which was planned as more of a refactoring PR first, but, welp, it's another story). 

#716 has nothing to do with LDAP in particular, we'd need RBAC subject validation. In `io.kafbat.ui.model.rbac.Permission` and `Role` classes there's a `validate` method, perhaps implementing one for `Subject` as well is the way here.

Reply:
Maybe we can merge this PR as is for the task fix https://github.com/kafbat/kafka-ui/issues/54 and I'll do the RBAC subject validation in a separate PR, what do you say?

Reply:
Let's rather not. This is not quite the changes we need to resolve #54, I've already started implementing the required adjustments in #717